### ISSUE TITLE
Smart fields, concatenate urls

### ISSUE THREAD

**Issue by daniel-costa** [ForestAdmin/forest-rails#170]
Hello,

I would like to concatenate multiple links in a smart field:

```
class Forest::Venue
  include ForestLiana::Collection

  collection :venues

  field :urls, type: 'String' do
    "<a href=\"#{object.facebook_url}\">FB</a><a href=\"#{object.twitter_url}\">TW</a>"
  end
end
```

- Any documentation available regarding the supported types for a smart field?
- Any chance it could support HTML markup, maybe a field type `HTML`?

**Comment by arnaudbesnier** [comment]
Hi @daniel-costa, yes you can do that with your Smart Field.

The only thing you have to do now, is to configure a "rich text editor" widget on your `urls` Smart Field in the `Venues` collection settings.
Once in "Edit Layout" mode in the UI go to Venues Settings > Fields > urls and define the "rich text editor" widget. You'll see your links properly displayed.

🌲🌲🌲

**Comment by daniel-costa** [comment]
Hi @arnaudbesnier 

I'm not getting that done right where I need it. 

#### Code
```ruby
class Forest::Venue
  include ForestLiana::Collection
  collection :venues
  field :urls, type: 'String' do
    # "<a href=\"www.test.com\">#{object.facebook_url}</a><a href=\"www.test.com\">#{object.twitter_url}</a>"
    "<strong>just a test</strong>"
  end
end
```
#### Field config
<img width="588" alt="screen shot 2017-08-03 at 2 49 00 pm" src="https://user-images.githubusercontent.com/10674268/28924871-0d052412-785b-11e7-995c-d119d10d62e0.png">

#### Collection
<img width="215" alt="screen shot 2017-08-03 at 2 49 18 pm" src="https://user-images.githubusercontent.com/10674268/28924882-16dde244-785b-11e7-89b5-4ea04b273f0b.png">

#### Details
<img width="116" alt="screen shot 2017-08-03 at 2 49 35 pm" src="https://user-images.githubusercontent.com/10674268/28924902-27096968-785b-11e7-9b3b-1ead8ca97530.png">

**Comment by arnaudbesnier** [comment]
@daniel-costa, so it works well in the details.
About the collection list, yes it is not well formatted, we could improve that.

For your business operations, Do you want to click on the links from the collection list?

**Comment by daniel-costa** [comment]
Yes

In fact, I think this might be a recurring problem. A user profile with multiple links (social links perhaps) and we want to display al of them in a "HTML icons" way.

For me, that field is relevant only on the collections view as a space saver. It allows me to see really fast if a link is missing if the icon doesn't appear.

**Comment by arnaudbesnier** [comment]
@daniel-costa we'll try to fix this in the current sprint then!
I'll let you know once it is fixed in production.

**Comment by daniel-costa** [comment]
Any news about this issue?

**Comment by daniel-costa** [comment]
Any news about this issue?

**Comment by arnaudbesnier** [comment]
Hi @daniel-costa, we have a lot of things to do right now so I can't give you any ETA for such a fix.

But I am sure you can find some workaround to see quickly in the list which social links are present or not (several Smart Fields for each social link, String Smart Field to concatenate the present social network names that are present, ...).

**Comment by daniel-costa** [comment]
Yes sure I could find workarounds, but it wasn’t the purpose when I posted the problem...
